targets/XPlatCppSdk/source/test/TestApp/TestContext.cpp

```diff
@@ -8,6 +8,29 @@ namespace PlayFabUnit
 {
     void TestContext::EndTest(TestFinishState state, std::string resultMsg)
     {
+        if (finishState != TestFinishState::PENDING)
+        {
+            resultMsg += testResultMsg;
+            resultMsg += "You can't mark a test as finished twice.\n";
+            if (finishState == TestFinishState::PASSED)
```

Comment:
This are the cases where we want to handle of sending a new message or adding times.